src/Compute/Compute/StorageServices/AddAzureVhdCommand.cs

```diff
@@ -228,13 +238,19 @@ public override void ExecuteCmdlet()
 
                         // 3-4: UPLOAD                  
                         WriteVerbose("Preparing for Upload");
-                        PSPageBlobClient managedDisk = new PSPageBlobClient(sasUri);
+                        ComputeTokenCredential token = null;
+                        if (this.DataAccessAuthMode == "AzureActiveDirectory")
+                        {
+                            // get token 
+                            token = new ComputeTokenCredential(DefaultContext, "https://disk.compute.azure.com/");
```

Comment:
please register this endpoint in azure environment

Reply:
> please register this endpoint in azure environment

what do you mean by that/ how do i do that?